pkg/scheduler/scheduler.go

```diff
@@ -384,6 +380,9 @@ func (sched *Scheduler) bind(assumed *v1.Pod, b *v1.Binding) error {
 		})
 		return err
 	}
+	if err := sched.config.SchedulerCache.FinishBinding(assumed); err != nil {
```

Comment:
`FinishBinding` will update field to trigger assumedPod cleanup , this PR will make assumePod here withouth cleanup.

Reply:
Why is it reasonable to attempt `FinishBinding` if `Bind` returns an err?

Reply:
> Why is it reasonable to attempt FinishBinding if Bind returns an err?

`FinishBinding` set ttl for assumedPod to cleanup.